code/modules/reagents/chemistry/recipes/toxins.dm

```diff
@@ -405,3 +405,151 @@
 	purity_min = 0.4
 	reaction_flags = REACTION_PH_VOL_CONSTANT
 	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_DAMAGING | REACTION_TAG_OTHER
+
+/datum/chemical_reaction/aus
+	results = list(/datum/reagent/toxin/aus = 6)
+	required_reagents = list(/datum/reagent/drug/space_drugs = 4, /datum/reagent/consumable/ethanol = 2, /datum/reagent/lithium = 2)
```

Comment:
this seems a bit too easy for a brain damage chem, especially considering it changes your name.

Reply:
ok, changed space drugs to ephedrine